### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by OlegHahm**
I just realized that the README is pretty outdated. This PR fixes at least the most obvious errors. More input is appreciated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,21 +35,23 @@ RIOT OS is an operating system for Internet of Things (IoT) devices. It is based
...
 * flexible memory management
-* high resolution timers
-* virtual, long-term timers
+* high resolution, long-term timers
+* support for AVR, MSP430, ARM7, and ARM Cortex-M on over 50 boards
 * the native port allows to run RIOT as-is on Linux, BSD, and MacOS. Multiple instances of RIOT running on a single machine can also be interconnected via a simple virtual Ethernet bridge
 * Wiselib support (C++ algorithm library, including routing, clustering, timesync, localization, security and more algorithms)
 * IPv6
-* UDP
 * 6LoWPAN
```
Comment (cgundogan) [inline README.md]: `6LoWPAN (IP and Next Header Compression)` ?

File: README.md [hunk 2]
```diff
@@ -35,21 +35,23 @@ RIOT OS is an operating system for Internet of Things (IoT) devices. It is based
...
-* NHDP
+* UDP
+* RPL
```
Comment (cgundogan) [inline README.md]: `RPL (Storing Mode)`

File: README.md (hunk 1 above)
Comment (OlegHahm) [inline README.md]: What about `(RFC4944, RFC6282, and RFC6775)` instead? Too cryptic?

File: README.md (hunk 2 above)
Comment (OlegHahm) [inline README.md]: Then we have to update as soon as we add other MOPs. Will we be aware of that?

Comment (cgundogan) [inline README.md]: >Will we be aware of that?

We `MUST` be aware of that. Better than giving false hopes (:

Comment (OlegHahm) [inline README.md]: addressed

File: README.md [hunk 3]
```diff
@@ -35,21 +35,22 @@ RIOT OS is an operating system for Internet of Things (IoT) devices. It is based
 Its features comprise
 * a preemptive, tickless scheduler with priorities
 * flexible memory management
-* high resolution timers
-* virtual, long-term timers
+* high resolution, long-term timers
+* support for AVR, MSP430, ARM7, and ARM Cortex-M on over 50 boards
```
Comment (cgundogan) [inline README.md]: I am just wondering: Should we change `AVR` to `AVR (8/16-bit)` ?

Comment (OlegHahm) [inline README.md]: Do we support AVR 16-bit?

Comment (OlegHahm) [inline README.md]: In general, I would try to keep the README as generic as possible. It's more like a teaser IMO.

Comment (cgundogan) [inline README.md]: OK, then let's keep teasing.

**Comment by cgundogan** [comment]
ACK, please squash

**Comment by OlegHahm** [comment]
Squashed

File: README.md (hunk 3 above)
Comment (cgundogan) [inline README.md]: OK, then let's keep teasing.

**Comment by cgundogan** [comment]
aaaaand GO!

**Pull request merged**